specification/monitor/Ingestion/routes.tsp

```diff
@@ -0,0 +1,48 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/rest";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
+
+namespace LogsIngestionClient;
+
+alias ServiceTraits = SupportsClientRequestId &
+  NoConditionalRequests &
+  NoRepeatableRequests;
+
+/**
+ * Ingestion API used to directly ingest data using Data Collection Rules.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+@summary("Ingestion API used to directly ingest data using Data Collection Rules.")
```

Comment:
Nit: since this is the same as the comment above, should we just remove this?

Reply:
Do you know if not having summary would leave gaps in generation? Was basing this off some of the existing typespec files I see with both the `@doc` and `@summary` decorators having the same strings. (Also, is my understanding correct that the `/** */` comments are equivalent to the `@doc` decorator?

Reply:
I dont believe so, but you can keep it if you want

Reply:
And yes /** */ is the same as the doc decorator